src/style.css

```diff
@@ -0,0 +1,57 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
don't reset margins and paddings with *-selector, do it for each element you need to reset